src/main/java/de/retest/recheck/ui/diff/ElementDifference.java

```diff
@@ -114,11 +114,11 @@ public String getIdentifier() {
 	}
 
 	private static String getSumIdentifier( final Collection<ElementDifference> differences ) {
-		String result = "";
+		final StringBuilder result = new StringBuilder();
 		for ( final ElementDifference difference : differences ) {
-			result += " # " + difference.getIdentifier();
+			result.append( "#" + difference.getIdentifier() );
```

Comment:
There is actually a space between `"_#"`. You could also use a stream 😄, similarly to the below (may need adjustment)
```
return differences.stream() //
    .map( ElementDifference::getIdentifier ) //
    .collect( Collectors.collectingAndThen( Collectors.joining( " #" ), ChecksumCalculator.getInstance()::sha256 )
```

The same goes for the others below.